Source/WebKit/UIProcess/ios/WKContentViewInteraction.h

```diff
@@ -566,6 +566,8 @@ struct ImageAnalysisContextMenuActionData {
 
     BOOL _keyboardDidRequestDismissal;
     BOOL _isKeyboardScrollingAnimationRunning;
+    BOOL _keyboardDismissedInCurrentPresentationUpdate;
+    BOOL _editingEndedWithReason;
```

Comment:
Nit - perhaps this would be more clear? (i.e. editing is ending due to user-triggered actions)

```suggestion
    BOOL _editingEndedByUser;
```